.circleci/config.yml

```diff
@@ -1,4 +1,4 @@
-version: 2
```

Comment:
This is the circleci config schema version, not the cache version. The cache version is `-v1-` here:

https://github.com/instacart/lore/pull/142/files#diff-1d37e48f9ceff6d8030570cd36286a61L32

          - source-v1-{{ .Branch }}-{{ .Revision }}

Reply:
@montanalow tests passed. i had to refactor the tests. separated out keras and other tests.
the issue was a memory problem due to statically linked libs (static TLS). while i don't fully understand the issue, too many imports would cause it to run out of memory slots